cloudera-integration/parcel/meta/parcel.json

```diff
@@ -7,7 +7,7 @@
     "baseVersion": "{{ version }}",
     "patchCount": "0"
   },
-  "depends": "CDH (>= 5.2), CDH (<< 6.0)",
+  "depends": "CDH (>= 5.2), CDH (<< 7.0)",
```

Comment:
Make both parcels work with either version of CDH. It's up to the admin to pick the right one. Maybe this can be fixed later